src/Adapter/OrderState/CommandHandler/EditOrderStateHandler.php

```diff
@@ -107,6 +107,10 @@ private function updateOrderStateWithCommandData(OrderState $orderState, EditOrd
 
         if (null !== $command->isSendEmailEnabled()) {
             $orderState->send_email = $command->isSendEmailEnabled();
+
+            if ($orderState->send_email && null !== $command->getTemplate()) {
```

Comment:
Same question as in the add handler, when `send_email` is forced to false, should the `$orderState->template` be forced to empty array to clean it?